lua/nvim-tree/populate.lua

```diff
@@ -73,6 +73,7 @@ local function link_new(cwd, name)
     absolute_path = absolute_path,
     link_to = link_to,
     open = open,
+    group_next = nil,   -- If node is grouped, this points to the next child dir/link node
```

Comment:
defining to nil is not required though, if trying to access group_next, it will be nil by default when not declared